composer.json

```diff
@@ -41,7 +18,7 @@
     "wpackagist-plugin/pantheon-advanced-page-cache": "^0.1.4",
     "wpackagist-plugin/wp-cfm": "^1.4.5",
     "wpackagist-theme/twentyseventeen": "^1.1",
-    "pantheon-systems/wordpress": "^4.8",
+    "pantheon-systems/wordpress-composer": "dev-master",
```

Comment:
@greg-1-anderson I respectfully disagree. When someone either builds a new site from this repo or runs `composer update` they should get the latest version.

If we use `^4.8` then `4.9` will not be used for new projects until we update the version here.